backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHContextApi.scala

```diff
@@ -39,15 +39,22 @@ import java.util.TimeZone
 
 class CHContextApi extends ContextApi with Logging {
 
-  override def initialize(conf: SparkConf): Unit = {
+  override def initialize(conf: SparkConf, isDriver: Boolean): Unit = {
     val libPath = conf.get(GlutenConfig.GLUTEN_LIB_PATH, StringUtils.EMPTY)
     if (StringUtils.isBlank(libPath)) {
       throw new IllegalArgumentException(
         "Please set spark.gluten.sql.columnar.libpath to enable clickhouse backend")
     }
-    // Path based load. Ignore all other loadees.
-    JniLibLoader.loadFromPath(libPath, true)
-
+    if (isDriver) {
+      JniLibLoader.loadFromPath(libPath, true)
+    } else {
+      val executorLibPath = conf.get(GlutenConfig.GLUTEN_EXECUTOR_LIB_PATH, StringUtils.EMPTY)
```

Comment:
use `libPath` as the default value, and directly call `loadFromPath(executorLibPath, true)`